docs/atomic-structure.md

```diff
@@ -236,17 +218,17 @@ You can get more information by reading the following:
 [A Simple Challenge to Classical Inheritance Fans](https://medium.com/javascript-scene/a-simple-challenge-to-classical-inheritance-fans-e78c2cf5eead#.mtrvhcjiw)
 [Composition over Inheritance (Youtube)](https://www.youtube.com/watch?v=wfMtDGfHWpA)
 
-#### Code and Related Pull Requests
+#### JavaScript component examples
 
 [View Unit Test](https://github.com/cfpb/cfgov-refresh/pull/916)
 [Expandable example 1](http://jsfiddle.net/0j9u66h0/9/)
 [Expandable example 2](https://jsfiddle.net/cpsyLy3L/2/)
 
-## Component Build Pipeline
+## Component build pipeline
```

Comment:
There are lots of other headings on this page that could also be sentence-cased.